internal/handlers/telegram/handler.go

```diff
@@ -83,9 +90,13 @@ stickerHandler handles the Message.Sticker Telegram Object, which formats the
 Telegram message into its base Emoji unicode character.
 */
 func stickerHandler(tg *Client, u tgbotapi.Update) {
-	formatted := tg.Settings.Prefix + u.Message.From.UserName +
-		tg.Settings.Suffix + u.Message.Sticker.Emoji
 
+	formatted := fmt.Sprintf("%s%s%s %s",
+		tg.Settings.Prefix,
+		GetUsername(u.Message.From),
+		tg.Settings.Suffix,
+		// Trim unexpected trailing whitespace
+		strings.Trim(u.Message.Sticker.Emoji, " "))
```

Comment:
Honestly just not sure why there would even be a trailing whitespace char here, but doesn't hurt

Reply:
I think its the library I am using in testing to convert strings to emoji. They must have add a space at the end

Reply:
fixed